README.rst

```diff
@@ -157,16 +157,16 @@ Using this SDK you can create steps, chain them together to create a workflow,
 create that workflow in AWS Step Functions, and execute the workflow in the
 AWS cloud.
 
-.. image:: https://github.com/aws/aws-step-functions-data-science-sdk-python/raw/master/doc/images/create.png
-  :scale: 50 %
-  :alt: Create a workflow in AWS Step Functions
+.. image:: https://raw.githubusercontent.com/aws/aws-step-functions-data-science-sdk-python/master/doc/images/create.png
```

Comment:
Good catch for broken images, but the raw URL isn't the problem. We switched branches from `master` -> `main` to use more inclusive language. It's GitHub's new default branch name now.

Can you use a relative path instead? That way it will also work between branches.
```suggestion
.. image:: ./doc/images/create.png
```

Reply:
Yes I can use a relative path. Will update it soon